pvr.iptvsimple/addon.xml.in

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <addon
   id="pvr.iptvsimple"
-  version="22.4.2"
+  version="22.4.3"
```

Comment:
That's the micro number. We want to change the mini number for a new feature. The correct version would be 22.5.0.